go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/prometheus/client_model v0.3.0 // indirect
 	github.com/prometheus/common v0.42.0 // indirect
 	github.com/prometheus/procfs v0.9.0 // indirect
+	github.com/rogpeppe/go-internal v1.10.0 // indirect
```

Comment:
Why was this library added as a dependency?

Can you run `go mod tidy` and see if that modifies `go.mod`. If it remains the same, then no additional commit is required.

Reply:
Fyi Please find the below screenshot :
![image](https://user-images.githubusercontent.com/99889761/233028564-d232ccec-796d-423e-bc43-9833e8ecf0e0.png)